src/CtaLink/CtaLink.levers.js

```diff
@@ -0,0 +1,38 @@
+// Levers shared by both CTA Links and CTA Link Button components.
+// ---------------------------------------------------------------------------
+
+/**
+ * Display type
+ */
+export const ctaLinkDisplay = ['inline-block', 'block', 'flex'];
+
+/**
+ * Icon options
+ */
+export const ctaLinkIcons = ['action', 'more', 'external', 'lock', 'download', 'video', 'jump', 'email', 'none'];
+
+/**
+ * Icon animation options
+ */
+export const ctaLinkAnimations = ['right', 'top-right', 'down', 'none'];
```

Comment:
```suggestion
export const ctaLinkAnimations = ['right', 'top-right', 'down'];
```

Reply:
I do find it useful to have a "none" option - since in Storybook, when you're using the controls to switch between options, it won't let you have an "unselected" or unset state once you pick something. So having this allow the user to unset the selection. I agree that if it's not because of Storybook, I'd just leave it out.